packages/calling/src/CallingClient/CallingClient.ts

```diff
@@ -504,6 +435,40 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public getLines(): Record<string, ILine> {
     return this.lineDict;
   }
+
+  /**
+   *
+   */
+  public getActiveCalls(): Record<string, ICall[]> {
+    const activeCalls = {};
+    const allCalls = this.callManager.getActiveCalls();
+    Object.keys(this.lineDict).forEach((deviceId) => {
+      const {lineId} = this.lineDict[deviceId];
+      activeCalls[lineId] = [];
+      Object.keys(allCalls).forEach((correlationId) => {
+        if (allCalls[correlationId].getLineId() === lineId) {
+          activeCalls[lineId].push(allCalls[correlationId]);
+        }
+      });
+    });
+
+    return activeCalls;
+  }
+
+  /**
+   *
+   */
+  public getConnectedCall(): ICall | undefined {
```

Comment:
same here need comments 

do you think its better to take line id as a parameter

Reply:
@arun3528 The purpose of creating this API was that if the user wants to fetch the information regarding which call is in a connected state at any point in time, they can call this API and get the call object which will have lineId as well.
Now they have this information on which line which call is in the connected state. Hence we need not provide lineId as a parameter here.

If we take lineId as a parameter here then the user has to keep providing each lineId one by one, calling this API multiple times and find which call is in the connected state.

Reply:
Added description